### ISSUE TITLE
Rising edge trigger doesn't work properly

### ISSUE THREAD

**Issue by tgtakaoka** [saleae/simple-parallel-analyzer#11]
Logic version: 2.4.6

The rising edge trigger of Simple Parallel anayzer doesn't work as expected (see screenshot).

It samples data (AD0~7) at falling edge of trigger signal (#RD and #WR), and the UI shows rising arrow mark on falling edge.

![image](https://user-images.githubusercontent.com/237988/219921647-19c267e7-68f7-4efd-b7c5-5c524d3f5e7a.png)

**Comment by tgtakaoka** [comment]
Probably fixed by dc343bca72d213c2b3b4754660cea35ef9a295f5, waiting for the fix is published via Logic update.

**Comment by timreyes** [comment]
Yes, sorry for the bug! We plan on implementing the fix in the next release of the Logic 2 software (2.4.7).
